lib/ebaze_web/templates/user/form.html.eex

```diff
@@ -10,7 +10,7 @@
   <%= error_tag f, :username %>
 
   <%= label f, :password %>
-  <%= text_input f, :password %>
+  <%= password_input f, :password %>
```

Comment:
card idea: for new user sign-up, have a confirmation field

Reply:
https://trello.com/c/NtnlLdjO